### PR TITLE
Update Speaking page for 2023

### PR DESCRIPTION
**Pull request by drewbrew**
1. Updates copy and links
2. Drops Josue from speaker mentors
3. Updates Katie's bio

Screenshots:
image
image
image
image
image

### CODE REVIEW

File: _pages/speaking-resources.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: Speaker Resources
 ---
 
 ## Information for Speakers
-* The Speaker Green Room will be in <strong>Santa Fe 4</strong> on Sunday-Wednesday.
+{% comment %}* The Speaker Green Room will be in <strong>Santa Fe 4</strong> on Sunday-Wednesday.{% endcomment %}
 * Please be in your presentation room at least 15 minutes before your talk to meet your session chair.
 * The Quiet Room will be in the <strong>Santa Fe 4</strong> on all conference days.
```
Comment (emullaney) [inline _pages/speaking-resources.md:12]: comment out Santa Fe 4 in line 12 as well?

**Review by emullaney — CHANGES_REQUESTED** [review]
Tried the links, read the copy. It looks good to me. I caught one mention of a San Diego room that needs to be removed.

File: _pages/speaking-resources.md (hunk 1 above)
Comment (drewbrew) [reply _pages/speaking-resources.md:12]: Good catch, thanks!

I'm going to change it to something like "A Quiet Room will be available on all conference days"

Comment (drewbrew) [reply _pages/speaking-resources.md]: Done

**Review by emullaney — APPROVED** [review]
LGTM!

**Pull request merged**